content/postgresql/postgresql-administration/create-superuser-postgresql.md

```diff
@@ -50,13 +50,13 @@ PASSWORD 'moreSecurePass';
 
 Third, verify the user creation:
 
-```sql
+```text
```

Comment:
why text? isn't it s a SQL query?

Reply:
Seems like the SQL syntax accounts for backslash commands but I agree with @IAmJSD this is psql syntax not part of SQL. Let's keep it text